Makefile

```diff
@@ -1,8 +1,12 @@
 SHELL=/bin/sh -e
 // Add list of charts: "chart1" "chart2" ...
 CHARTS := "spring-boot-example-app"
+SNOWDROP_GITHUB_ORG := "https://github.com/snowdrop"
+REMOTE_CHARTS := "cache-example" "circuit-breaker-example" "configmap-example" "crud-example" "health-check-example" "messaging-work-queue-example" "rest-http-example"
```

Comment:
Does it make sense to hard code here the examples instead of passing a list when project is built using maybe a github query to get all the projects having as label `helm`?

Reply:
I would prefer keep it simple for now.